Added resets and variables

moonclash opened the pull request:
Added resets for links and lists, variables for colors and paddings

Commit:
Added resets and variables

Added resets for links and lists, variables for colors and paddings

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+/* Resets */
+
```
Add 

``` css
* {
  box-sizing: border-box;
}

```

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+/* Resets */
+
+ul {
+
```
Remove empty line here, not needed

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+}
+
+a {
+
```
Remove empty line here not needed

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+a {
+
+  text-decoration: none;
+  color: #fff;
```
You know because the default background is white like pure white I suggest using another color for the links by default. Even though our team is dark. You could look into another white-ish color. Look at material design colors in google and find a white-ish color if you want to stick with white but I think you should change it to a darker color.

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+}
+
+
+/* Colors */
```
Move all the variables to `variables.scss` in Base :)
P.S If you don't have the variables file or its not visible compare with master and update from it aka pull master :)

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+
+/* Colors */
+
+$main-black: #1F1C17;
```
No need for the `main-` prefix. Just use `$black` cause thats OUR "black" color ;)
Do the same for the other variables.

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+/* Resets */
```
Rename this file to be `default.scss` not `theme.scss`

william-abboud commented on src/styles/Base/theme.scss:
```diff
@@ -0,0 +1,38 @@
+
+$main-white: #FAFAFA;
+
+/* Paddings */
```
I say remove the padding styles you've added for now until we actually decide on the vertical rythm of things I think its very difficult to tell from now

Commit:
Initial edits

Added initial edits as per Will

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into theme-colors

Commit:
Added changes

Made changes as per Will suggestions. Moved variables and moved them to
the variable folder, renamed theme.scss to default.scss. Removed
paddings variables.

william-abboud commented on src/styles/Base/default.scss:
```diff
@@ -0,0 +1,25 @@
+  text-decoration: none;
+  color: $black;
+}
+
```
WoW don't add these empty lines between line 16 and 25

Commit:
Deleted old theme file

Commit:
Removed white space

Commit:
Added default.scss to main.scss

william-abboud commented:
Awesome work :shipit: 
P.S We need to discuss variations of the main colors. Sometimes you won't need that exact `yellow` but some other variation of that yellow maybe a darker one for a border for example. Make a note somewhere when you make the design that we have to address that.

:ship: :dart: :hankey:
